### PR TITLE
fetchUsefuls

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -19,8 +19,12 @@ app.get('^/:id(' + articleUuidRegex + ')/people', require('./controllers/related
 app.get('^/:id(' + articleUuidRegex + ')/organisations', require('./controllers/related/organisations'));
 app.get('^/:id(' + articleUuidRegex + ')/comments-hack', require('./controllers/comments-hack'));
 
+
+// temporary (ma)
+app.get('/more-on/useful', require('./controllers/useful'));
```
Comment (ironsidevsquincy) [inline server/app.js]: was wondering if these routes should live under the article itself, like people/organisations/comments-hack, as they are related to the article itself, and doesn't pollute the top level, e.g.

`/7f473d26-be68-11e4-8036-00144feab7de/useful` (POST and GET)
`/7f473d26-be68-11e4-8036-00144feab7de/related`
`/7f473d26-be68-11e4-8036-00144feab7de/more-on?relation=primaryTheme`

**Comment by ironsidevsquincy** [comment]
So this is a kind of crowd-sourced popular?

File: server/app.js (hunk 1 above)
Comment (matthew-andrews) [inline server/app.js]: at the moment this is global

**Comment by matthew-andrews** [comment]
it's basically recommended, repacked.

**Pull request merged**